compiler/src/dotty/tools/dotc/tastyreflect/ReflectionCompilerInterface.scala

```diff
@@ -1245,6 +1245,10 @@ class ReflectionCompilerInterface(val rootContext: core.Contexts.Context) extend
 
   def TypeRef_name(self: TypeRef)(given Context): String = self.name.toString
 
+  def TypeRef_isOpaqueAlias(self: TypeRef)(given Context): Boolean = self.symbol.isOpaqueAlias
+
+  def TypeRef_opaqueAliasWrappedType(self: TypeRef)(given Context): Type = self.translucentSuperType
```

Comment:
It might be better to keep the same method name. Changing names has shown to lead to confusion in the past.